lesson_12/lesson_12.cpp

```diff
@@ -0,0 +1,112 @@
+﻿#include <iostream>
+#include <iomanip>
+
+#include "arrays.hpp"
+#include "console.hpp"
+
+#include "headers/sort_utils.hpp"
+
+void print_menu();
+
+void bubble_sort();
+void q_sort();
+
+
+int main()
+{
+    std::srand(std::time(nullptr));
+    std::cout << "Home Work 12\n";
+    std::cout << "-----------\n\n";
+
+    int user_choice{ 0 };
+    while (user_choice != 3)
+    {
+        print_menu();
+        std::cin >> user_choice;
+        utl::clear_console();
+        switch (user_choice)
+        {
+        case 1:
+            bubble_sort();
+            break;
+        case 2:
+            q_sort();
+            break;
+        case 3:
+            std::cout << "Exit\n";
+            break;
+        default:
+            std::cout << "Wrong choice!";
+            user_choice = 0;
+            break;
+        }
+        utl::wait_to_continue();
+    }
+
+    return 0;
+}
+
+
+void print_menu()
+{
+    std::cout << "1. bubble sort\n";
+    std::cout << "2. quick sort\n";
+    std::cout << "3. Exit\n\n";
+
+    std::cout << "Enter number:";
+}
+
+void bubble_sort()
```

Comment:
it's better to chreate some other name for this function

Reply:
done